Backpack-SwiftUI/ImageGalleryGrid/Classes/ImageGalleryGridContentView.swift

```diff
@@ -0,0 +1,84 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+struct ImageGalleryGridContentView<ImageView: View>: View {
+    
+    let categories: BPKImageGalleryCategoriesTypeContainer<ImageView>
+    let closeAccessibilityLabel: String
+    let onCategoryChanged: () -> Void
+    let itemTapped: (BPKImageGalleryImage<ImageView>) -> Void
+    let onCloseTapped: () -> Void
+    
+    @Binding var selectedCategoryIndex: Int?
```

Comment:
As above, is it really possible to have a nil selection for this?

Reply:
✅